src/app/components/task-list/task-list.component.ts

```diff
@@ -16,20 +16,15 @@ import { RemoveButtonComponent } from 'src/app/remove-button/remove-button.compo
 })
 export class TaskListComponent implements OnInit, OnDestroy {
   completedTasksSubscription = Subscription.EMPTY;
-  exampleTasksSubscription = Subscription.EMPTY;
   tasksSubscription = Subscription.EMPTY;
   tasksCompleted: number = 0;
   tasks: Task[] = [];
 
   constructor(private http: HttpClient, private taskService: TaskService) {}
 
   ngOnInit(): void {
-    this.exampleTasksSubscription = this.taskService.getExampleTasks().subscribe((tasks) => {
-      if(!this.taskService.initialized){
-        this.taskService.setTasks(tasks);
-      } 
-    })
-    this.tasksSubscription = this.taskService.tasksSubject.subscribe((tasks) => {
+    this.tasksSubscription = this.taskService.getTasks().subscribe((tasks) => {
```

Comment:
also if you only intend to call this method once to fetch the task list then you can try using take operator from rxjs and take only 1 value coming from the subscribe:
getTasks().pipe(take(1)).subscribe(tasks => etc etc);

Reply:
this way you wont need to declare a subscription for this method as it will automatically unsubscribe after taking 1 value